policyengine_us/parameters/gov/states/ri/tax/income/exemption/phase_out/start.yaml

```diff
@@ -0,0 +1,16 @@
+description: Rhode Island phases the exemption out starting at this income threshold.
+values:
+  2021-01-01: 210_750
+  2022-01-01: 217_050
```

Comment:
@MaxGhenis the [law](http://webserver.rilin.state.ri.us/Statutes/TITLE44/44-30/44-I/44-30-2.6.htm) provides a breakdown by filing status but the [tax form](https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022_1040WE_w_0.pdf) does not for either the income threshold or the multiplier (which is halved for separate filers in the law) do, would the form be more applicable for us?

Reply:
@mylittlebecca after speaking with Max, we can model the form we should file an issue saying that the law does not overlap with the form, but we will model the worksheet - please check if there is a worksheet for separate filers